### DISCUSSION TITLE
Jekyll/Hyde mutation(s)

### DISCUSSION THREAD

**Discussion by actual-nh** [CleverRaven/Cataclysm-DDA#45946] [Q&A] [answered]
Hi. I've been wondering about this pairing:

- What is the meaning of them having different point costs, when the character switches back and forth between them?
- Why is this not a BEAST category mutation (as well as MEDICAL)?

BTW, to the developers: Thanks for a very interesting game! (Admittedly, as usual for me with computer games, I'm more interested in the world and coding than in playing...)

**Comment by anothersimulacrum** [comment, marked as answer]
Introduced in #40803

>  What is the meaning of them having different point costs, when the character switches back and forth between them?

This appears to be to change the color in the trait display.

> Why is this not a BEAST category mutation (as well as MEDICAL)?

Presumably because it fits better with the theme of Jekyll/Hyde - it was introduced by intaking a serum, something that I feel fits better with medical than with beast. You're a normal human transformed by something you have created, not just a beast.
